.github/workflows/publish-to-pypi.yaml

```diff
@@ -0,0 +1,37 @@
+name: Push Python distributions to PyPI
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+# on:
```

Comment:
do we need this?

Reply:
First set the trigger condition to debug when PR occurs, then use the commented code (tag trigger) when it is OK